### PR TITLE
Update 1906-09-17.xml

### PR DESCRIPTION
**Pull request by jimmymendez123**

### CODE REVIEW

**Comment by whanley** [comment]
I'm confused. There's a lot of material here from the following week (September 22 1906). How come?

**Comment by whanley** [comment]
There's a lot of material in this issue, and it's quite well structured, but I can't understand why it's from September 22.

File: 1906-09-17.xml [hunk 1]
```diff
@@ -1441,8 +1441,8 @@
             <pb n="2"/>
```
Comment (huoyanzhizhou) [inline 1906-09-17.xml]: Where is the page div?

File: 1906-09-17.xml [hunk 2]
```diff
@@ -1441,16 +1439,4536 @@
...
+                        believe him to be, a credible witness, the authorities there will do what
+                        they can to hasten the development, and not only in Egypt, but also in
+                            <placeName>India</placeName>. </p>
+                </div>
             <pb n="3"/>
-            
+                        <div type="page" n="3">
+                <div type="section" feature="local">
+                    <head>Local and General </head>
+                    <p>Italian Cruiser </p>
```
Comment (huoyanzhizhou) [inline 1906-09-17.xml]: please make small individual stories in a section into individual items.

File: 1906-09-17.xml [hunk 3]
```diff
@@ -1441,16 +1439,4536 @@
...
+                        contractors, who have provided in the insurance contract that the yacht will
+                        be navigated as far as <placeName>Gibraltar</placeName> in the charge of a
+                        special pilot. Seeing that <name> the Mabroussa</name> was originally built
+                        of iron on the <placeName>Thames</placeName> as long ago as 1865, some
+                        underwriters have been surprised that she was worth the enormous amount
+                        spent upon her reconstruction, but they are unanimous in considering her a
+                        first-class risk for the voyage out, as she is practically an entirely new
+                        yacht, inferior probably to no other pleasure craft of the same type afloat.
+                            <name>The Mabroussa</name> is not the first yacht built for a
+
```
Comment (huoyanzhizhou) [inline 1906-09-17.xml]: I am also confused to see wires of 1906-09-22 to show up in a 1906-09-17's newspaper. Is this the mistake of the newspaper itself?

**Comment by whanley** [comment]
What's happening with your deliverables @jimmymendez123?

**Comment by whanley** [comment]
There are some issues here--especially with OCRed material pasted without paragraphs--but I'll merge these anyway.

**Pull request merged**